conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
+Alias /lsmbcss /var/lib/master/css/
+
+<VirtualHost *:80>
+
+  ServerName [SERVERNAME]:80
+  DocumentRoot WORKING_DIR
+
+  <Proxy *>
+    Order Allow,Deny
+    Allow from All
+  </Proxy>
+
+  LogLevel Warn
+
+  <FilesMatch ...>
+    SetHandler  "proxy:pl://localhost:5001"
```

Comment:
https://httpd.apache.org/docs/current/mod/core.html#sethandler

Le 2016-04-22 14:40, Erik Huelsmann a écrit :
>
> In conf/ledgersmb-apache-2.4.template
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60784488>:
>
> > +Alias /lsmbcss /var/lib/master/css/
> > +
> > +<VirtualHost *:80>
> > +
> > +  ServerName [SERVERNAME]:80
> > +  DocumentRoot WORKING_DIR
> > +
> > +  <Proxy *>
> > +    Order Allow,Deny
> > +    Allow from All
> > +  </Proxy>
> > +
> > +  LogLevel Warn
> > +
> > +  <FilesMatch ...>
> > +    SetHandler  "proxy:pl://localhost:5001"
>
> @ylavoie <https://github.com/ylavoie> can't find any reference to
> proxy:pl on the web. Do you have a reference for me?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60784488>
>